packages/loader/container-loader/src/containerContext.ts

```diff
@@ -57,8 +57,8 @@ export class ContainerContext implements IContainerContext {
 		quorum: IQuorum,
 		loader: ILoader,
 		submitFn: (type: MessageType, contents: any, batch: boolean, appData: any) => number,
-		submitSummaryFn: (summaryOp: ISummaryContent) => number,
-		submitBatchFn: (batch: IBatchMessage[]) => number,
+		submitSummaryFn: (summaryOp: ISummaryContent, referenceSequenceNumber?: number) => number,
```

Comment:
ContainerRuntime should add referenceSequenceNumber when sending summary message. That change is missing.

Reply:
Not sure if we can right now: https://github.com/microsoft/FluidFramework/pull/14150#discussion_r1106452120

I'd rather leave that to use the old flow for now, but have the loader ready to receive it when it's settled.

Reply:
If I understand correctly, I think Vlad is talking about a future where summaries can happen while inbound ops are flowing. However, today we can still send the referenceSequenceNumber when submitting the summary message. It would be the same as the one delta manager would add so not adding it will have the same behavior, but I think we should still do it for consistency.

Not doing it in this PR is fine, but we at least should track it.

Reply:
ah, I see. Yeah, it's an easy change in this case.